lib/stratum.js

```diff
@@ -64,6 +64,27 @@ var StratumClient = function(params){
                 _this.lastActivity = Date.now();
                 handleSubmit(message);
                 break;
+            case 'alph_submitHashrate':
+                break;
+            case 'mining.subscribe':
+                sendJson({
+                    id: message.id,
+                    result: "result",
+                    error: null
+                });
+                break;
+            case 'mining.authorize':
+                sendJson({
+                    id: message.id,
+                    result: true,\
+                    error: null
+                });
+                sendJson({
+                    method: "mining.set_extranonce",
+                    params: [(Math.floor(Math.random()*10000))],
```

Comment:
Will this param be used by miners?

Reply:
Should this extranonce be a hex string, like "af4c"?

Reply:
yes. (Math.floor(Math.random() * 65535).toString(16)) would work better there for larger randomness.